app/routes/topics/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const express = require('express');
+
+const getTopicsMiddleware = require('../../../lib/middleware/topics/index/topics-get');
+
+const router = express.Router({ mergeParams: true });
```

Comment:
I don't think `mergeParams` is needed here since it's an index route and the path won't include params.